Slight fixes: Grammar / Wording (#2)
andrechuakj/tp: docs/2101UserGuide.md

#### Enrol a Member to an Event: `enrol`

> Now that you have added all your members and all your events, keeping track of your members’ attendance at any event is as simple as using the `enrol` command!

Reviewer: Should we change attendance to enrolment here instead?
```suggestion
Now that you have added all your members and all your events, keeping track of your members’ enrolment to any event is as simple as using the `enrol` command!
```

<div markdown="block" class="alert alert-primary">:bulb:
The find commands are powerful tools to help you work faster. Want to view the attendance of Nicholas, but you don’t 

> want to scroll through the member list? Use the `findMember` then follow it with the `viewMember` command to achieve that. 

Reviewer: ```suggestion
want to scroll through the member list? Use the `findMember` command then follow it with the `viewMember` command to achieve that. 
```
